GOVERNANCE.md

```diff
@@ -185,6 +190,7 @@ Contributors don't need to be part of any dedicated team.
 * Bill Engels (dev)
```

Comment:
Nit: Austin is missing a label on line above

Reply:
it is perfectly fine to not be part of any team :smile:. Austin's current contributions don't really fit in any of the teams but that doesn't make them less important (we spoke on slack and he is 100% identified with "no team" team). 

If we had someone manage twitter and write grants but not work on on dev, docs nor answer on discourse they would also have no dedicted teams listed for example.

Reply:
Thanks!